src/org/zaproxy/zap/extension/bruteforce/BruteForceParam.java

```diff
@@ -164,13 +180,73 @@ public void setFileExtensions(String fileExtensions) {
 	    }
 	    
 	    List<String> fileExtensionsList = new ArrayList<>();
-	    for (String fileExtension: fileExtensions.replaceAll("\\s", 
-	    		EMPTY_STRING).split(",")) {
+	    for (String fileExtension: fileExtensions.replaceAll(WHITESPACE, 
+	    		EMPTY_STRING).split(COMMA)) {
 	    	if (!fileExtension.equals(EMPTY_STRING)) {
 	    		fileExtensionsList.add(fileExtension);
 	    	}
 	    }
 	    
     	return fileExtensionsList;	    
 	}
+
+	/**
+	 * @return {@code String} of comma-separated file-extensions that are not to be processed.
+	 * {@code "jpg, gif, jpeg, ico, tiff, png, bmp"} is returned by default

```

Comment:
Probably better to link to the constant instead, otherwise this will have to be changed if we happen to add/remove an extension.